src/watermarks/dto/update-watermark.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { PartialType } from '@nestjs/mapped-types';
```

Comment:
This file is never used, you can remove that.

Reply:
Done, deleted

Reply:
Now solve conflicts and merge to develop.